docs/openapi/api-internal-pn-bff-tos-privacy.yaml

```diff
@@ -60,11 +60,67 @@ paths:
       tags:
         - UserConsents
       operationId: acceptTosPrivacyV2
-#      security:                                      # ONLY EXTERNAL
-#        - bearerAuth: [ ]                            # ONLY EXTERNAL
+      #      security:                                      # ONLY EXTERNAL
+      #        - bearerAuth: [ ]                            # ONLY EXTERNAL
       parameters: # NO EXTERNAL
         - $ref: 'common-refs.yaml#/components/parameters/uidAuthFleet'               # NO EXTERNAL
         - $ref: 'common-refs.yaml#/components/parameters/cxTypeAuthFleet'            # NO EXTERNAL
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: './tos-privacy-schemas/tos-privacy.yaml#/components/schemas/BffTosPrivacyBody'
+      responses:
+        '200':
+          description: OK
+        '400':
+          description: Bad Request
+          content:
+            application/problem+json:
+              schema:
+                $ref: './common-refs.yaml#/components/schemas/Problem'
+  '/bff/v1/pg/tos-privacy':
+    get:
+      summary: Accettazione Pg TOS e Privacy
+      description: >-
+        Ritorna lo stato di accettazione dei TOS e della Privacy del Pg.
+      tags:
+        - UserConsents
+      operationId: getPgConsentByType
+      #      security:                                                                # ONLY EXTERNAL
+      #        - bearerAuth: [ ]                                                      # ONLY EXTERNAL
+      parameters:
+        - $ref: './common-refs.yaml#/components/parameters/uidAuthFleet'             # NO EXTERNAL
+        - $ref: './common-refs.yaml#/components/parameters/cxTypeAuthFleet'          # NO EXTERNAL
+        - $ref: './tos-privacy-schemas/tos-privacy.yaml#/components/parameters/BffQueryTosPrivacyType'
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: './tos-privacy-schemas/tos-privacy.yaml#/components/schemas/BffTosPrivacyConsent'
+        '400':
+          description: Bad Request
+          content:
+            application/problem+json:
+              schema:
+                $ref: './common-refs.yaml#/components/schemas/Problem'
+    put:
+      summary: Accettazione Pg TOS e Privacy
+      description: >-
+        Permette di accettare i TOS e la Privacy del Pg.
+      tags:
+        - UserConsents
+      operationId: acceptPgTosPrivacyV1
+      parameters:
```

Comment:
Because all the paramters have `# NO EXTERNAL` also the parameters key must to have it

Reply:
done